src/main/java/frc/team2412/robot/commands/intake/IntakeOutCommand.java

```diff
@@ -0,0 +1 @@
+
```

Comment:
Should this be put off to another PR? Is this supposed to be different from `IntakeSetOutCommand`?

Reply:
we need both we need IntakeOut for IntakeSetOut